### PR TITLE
Update Quickstart with Single File Analysis Demo

### PR DESCRIPTION
**Pull request by ryanohoro**
**Describe the change**

Updates the Quickstart in README.md to a workflow that demonstrates Strelka's capabilities with minimal command line work.

1. Install prerequisites
2. Download Strelka
3. Install yara rules
4. Build Strelka
5. Preparation of sample file
6. Use oneshot to analyze file and output event JSON
7. Explanation of the Strelka pipeline and its features
8. Sample events
9. Suggestion for detection opportunity 

Moves default yara file around to ease use.

Adds a oneshot docker service for use without go build, permits ENV vars for configuration. 

```
ENV STRELKA_ONESHOT_FRONTENDURL=frontend:57314
ENV STRELKA_ONESHOT_LOGPATH=-
```

**Describe testing procedures**

These instructions were tested on a fresh Ubuntu 22.04 VM.

Updated go and containers built successfully. Oneshot container and binary generated a Strelka event successfully.

**Sample output**

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of and tested my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,40 +24,170 @@ Strelka is a modular data scanning platform, allowing users or systems to submit
...
+
+#### Step 2: Download Strelka
+
+```bash
+git clone https://github.com/target/strelka.git && \
+cd strelka
 ```
 
-#### Step 2: Build [Strelka-Fileshot](https://github.com/target/strelka/blob/master/docs/README.md#strelka-fileshot) (File Submitter)
+#### Step 3: Download and install preferred yara rules (optional)
```
Comment (phutelmyer) [inline README.md:48]: Great callout here.

File: README.md [hunk 2]
```diff
@@ -24,40 +24,170 @@ Strelka is a modular data scanning platform, allowing users or systems to submit
...
-  ...
+
+#### Step 6: Analyze the file with Strelka using the dockerized oneshot
+
+```bash
+./strelka-oneshot -f samples/Win32.Emotet.zip -l - | jq
 ```
 
-#### Step 4: Run Strelka-Fileshot and Review Output
+#### What's happening here?
```
Comment (phutelmyer) [inline README.md:78]: 💯

File: go.work [hunk 3]
```diff
@@ -0,0 +1,7 @@
+go 1.19
```
Comment (phutelmyer) [inline go.work:1]: First I've seen a go workspace file, good add.

File: configs/python/backend/backend.yaml [hunk 4]
```diff
@@ -634,7 +634,7 @@ scanners:
           - '*'
       priority: 5
       options:
-        location: '/etc/yara/'
+        location: '/etc/strelka/yara/rules.yara'
```
Comment (phutelmyer) [inline configs/python/backend/backend.yaml:637]: I'm good with this, although I'll give this a specific callout in the CHANGELOG.

File: build/docker-compose.yaml [hunk 5]
```diff
@@ -55,6 +54,17 @@ services:
     depends_on:
       - coordinator
 
+  oneshot:
```
Comment (phutelmyer) [inline build/docker-compose.yaml:57]: As oneshot is a submission client, this shouldn't be in the compose file, right?

Comment (ryanohoro) [reply build/docker-compose.yaml:57]: You're probably right that it should have its own compose file, I'll make that change.

**Review by phutelmyer — APPROVED** [review]

**Pull request merged**